index.ts

```diff
@@ -104,8 +104,8 @@ export default function mem<
 		maxAge,
 	}: Options<FunctionToMemoize, CacheKeyType> = {},
 ): FunctionToMemoize {
-	if (typeof maxAge === 'number') {
-		mapAgeCleaner(cache as unknown as Map<CacheKeyType, ReturnType<FunctionToMemoize>>);
+	if (typeof maxAge === 'number' && maxAge <= 0) {
+		return fn;
```

Comment:
This feels a little icky. <0 should probably be an error. Who memoizes a function with a calculated maxAge?

Reply:
Agreed. Will do.

Reply:
Similar situation in https://github.com/Richienb/batched-function#delay

- delay: undefined -> trigger in next micro task
- delay: 0 -> trigger the same way `setTimeout(fn, 0)` would

Either way not pretty